lib/MusicBrainz/Server/Entity/URL/Gakki.pm

```diff
@@ -0,0 +1,23 @@
+package MusicBrainz::Server::Entity::URL::Gakki;
+
+use Moose;
+use utf8;
+
+extends 'MusicBrainz::Server::Entity::URL';
+with 'MusicBrainz::Server::Entity::URL::Sidebar';
+
+sub sidebar_name { '私家版楽器事典' }
```

Comment:
It seems to just mean “Private musical instrument encyclopedia”. Might be worth mentioning whose it is. The whole information is given in homepage’s footer.
```suggestion
sub sidebar_name { 'IROMBOOK 私家版楽器事典' }
```

Reply:
Apparently the actual meaning is more like "self-made" than "private", but still, that might make sense - if that's correct in Japanese. I'll ask a JP speaker.

Reply:
Confirmed that it's grammatical in Japanese, making the change :)